src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java

```diff
@@ -32,38 +36,46 @@
     @FindBy(id = "login_button")
     private ExtendedWebElement loginBtn;
 
+    final Wait<WebDriver> wait = new WebDriverWait(driver, 5);
+
     public LoginPage(WebDriver driver) {
         super(driver);
     }
 
     @Override
     public void typeName(String name) {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("name")));
```

Comment:
you can change 'By.id...' to 'element.getBy', because you create all these elements above

Reply:
done